gradle.properties

```diff
@@ -1,4 +1 @@
-# Sets default memory used for gradle commands. Can be overridden by user or command line properties.
-# This is required to provide enough memory for the Minecraft decompilation process.
-#org.gradle.jvmargs=-Xmx3G
-org.gradle.daemon=false
\ No newline at end of file
+org.gradle.jvmargs=-Xmx1G
```

Comment:
Will this be enough memory for Forge to decompile MC?

Reply:
as you can see from the previously-commented-out 3G arg, it became unnecessary ages ago when FG re-did their process. The issue is now with loom.

Reply:
Yes, this is more a consequence of the Gradle 5 update -- the daemon now uses `-Xmx64m` by default, rather than letting the VM pick max memory (which is usually ~1g). This value was determined by running the build until Gradle stopped printing memory errors.